### PR TITLE
feat: support overwrite flag on ABS workspaces

### PR DESCRIPTION
**Pull request by odinuv**
https://keboola.atlassian.net/browse/PS-2160

### CODE REVIEW

File: src/File/Strategy/Local.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
 
 class Local extends AbstractStrategy implements StrategyInterface
 {
-    public function downloadFile($fileInfo, $destinationPath)
+    public function downloadFile($fileInfo, $destinationPath, $overwrite)
```
Comment (pivnicek) [inline src/File/Strategy/Local.php:11]: just a note for future if we want ti implement this in `Local` we'll need to update the `writeFile` somehow.  Currently `dumpFile` has hard overwrite `true` 
https://github.com/symfony/symfony/blob/35dbf8c81fad376da61978d7ffd48b623522b97a/src/Symfony/Component/Filesystem/Filesystem.php#L668
Obviously that doesn't matter for jobs, but sandboxes may want that flexibility

File: tests/Functional/DownloadFilesTest.php [hunk 2]
```diff
@@ -766,11 +790,6 @@ public function testReadFilesForBranchWithProcessedTags()
         $this->clientWrapper->getBasicClient()->deleteFile($branchProcessedFileId);
     }
 
-    public function testReadFilesChangedSinceDateString()
-    {
-        $this->markTestIncomplete('TODO');
-    }
-
```
Comment (pivnicek) [inline tests/Functional/DownloadFilesTest.php:773]: 🤦 thanks!

**Review by pivnicek — APPROVED** [review]
sweet 👍 

I like how we're slowly fixing the quotes 😄

File: src/File/Strategy/Local.php (hunk 1 above)
Comment (odinuv) [reply src/File/Strategy/Local.php:11]: Yes, I was wondering whether to make this complete or not, but then we should make that option for databases workspaces too and that seemed to be a long journey.

Comment (pivnicek) [reply src/File/Strategy/Local.php:11]: yeah, I think we can put it off.  maybe we should throw a user exception if anyone tries `overwrite: false` on local?

Comment (odinuv) [reply src/File/Strategy/Local.php:12]: good idea, done that, so that it's not ending in vacuum

**Pull request merged**